server/controllers/userController.js

```diff
@@ -29,44 +30,59 @@ class UserController {
       phoneNumber,
       consealedPassword,
       type,
+      isAdmin,
     });
 
-    if (result === '23505') return response.error(res, 400, 'Email already exist');
+    if (result === '23505') {
```

Comment:
I'm not sure we should leave error handling to the database. We want to be able to control what happens in our app.

Reply:
Okay